kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/kms/vault/KubeVaultTestKmsFacadeFactory.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.resources.kms.vault;
```

Comment:
Certainly a thought for another day but a `resources` package feels a bit odd (I think I follow the naming) as resources in maven projects is ususually `src/[main|test]/resources/` and filled with properties files and yaml docs etc.

Reply:
This was inherited from the Strimzi system tests implementation.

Reply:
Issue #1351 created.